config/locales/en.yml

```diff
@@ -73,6 +76,18 @@ en:
     log_out: "Exit"
     logged_out: "Your session has been cleared."
     new: "Submit contact information"
+    new_auth_token: "Log in with email"
+    new_auth_token_header: "Existing publisher login"
+    new_auth_token_body: "If you've already started or completed verification, we can send a login link to the email address associated with your publisher registration."
+    new_auth_token_body_2: "If you've forgotten the email:"
+    new_auth_token_body_3: "1) For verifications completed, leave email blank and we'll use the most recent email."
+    new_auth_token_body_4: "2) For verifications in progress, please "
```

Comment:
With the comment above too, I think it's worth suggesting (as I did in the text) that you can provide the email and domain if the person has already started (but not finished) the process... BUT they can just start the process over again 😄   The wording makes the email field look useless since it's telling you to:
- not put email if finished
- if not finished, don't even use this- start the process over